frontend/src/views/PageNotFound.vue

```diff
@@ -0,0 +1,82 @@
+<template>
+  <v-container fluid>
+    <content-card
+      class="contentCard"
+      :title="$tc('views.pageNotFound.title')"
+      max-width="1200"
+      toolbar
+    >
+      <template #title-actions>
+        <v-btn
+          class="d-md-none"
+          icon
+          :to="{ name: 'profile', query: { isDetail: true } }"
+        >
+          <user-avatar :user="user" :size="36" />
+        </v-btn>
+      </template>
+
+      <v-card-text class="justify-center">
+        <v-img contain class="error-image" src="@/assets/FourZeroFour.svg"></v-img>
+        <p
+          id="error-title"
+          class="font-weight-bold text-center px-8 text-xs-h6 text-sm-h5 text-md-h4 text-lg-h3 text-xl-h1"
```

Comment:
```suggestion
          class="font-weight-bold text-center px-8 text-xs-h6 text-sm-h5 text-md-h4"
```
Larger is unnecessary and too big.